appshell/appshell_extensions_win.cpp

```diff
@@ -969,6 +969,7 @@ int32 ReadFile(ExtensionString filename, ExtensionString& encoding, std::string&
                                 detectedCharSet = conv.to_bytes(encoding);
                             }
                             if (detectedCharSet == "UTF-16LE" || detectedCharSet == "UTF-16BE") {
+				 CloseHandle(hFile);
```

Comment:
this looks good. Are there other places where File Handles need to be closed ? Like in catch blocks ?

Reply:
I think this is the only place. FYI, this implementation was changed recently in this [PR](https://github.com/adobe/brackets-shell/pull/635/files).